app/partials/class-list.html

```diff
@@ -1,7 +1,8 @@
 <div>
 <h2>Concept Class Management</h2>
+	<h3>{{(classAdded.length > 0) && (classAdded != ' ') ? classAdded + ' has been added' : ''}}</h3>
```

Comment:
That's a lot better :)
Fixed in: https://github.com/rkorytkowski/openmrs-owa-conceptdictionary/commit/27bc83a596aaaca2652f86e0da7f9f0bf3797a70